test/actions/closeVault.test.ts

```diff
@@ -0,0 +1,51 @@
+import { NATIVE_MINT } from '@solana/spl-token';
+import { Connection, NodeWallet } from '../../src';
+import { createVault, closeVault } from '../../src/actions';
+import { FEE_PAYER, pause, VAULT_EXTENRNAL_PRICE_ACCOUNT } from '../utils';
+import { mockAxios200 } from './shared';
+import { Vault } from '../../src/programs/vault';
+
+jest.mock('axios');
+
+describe('closing a Vault', () => {
+  const connection = new Connection('devnet');
+  const wallet = new NodeWallet(FEE_PAYER);
+
+  // console.log(FEE_PAYER.publicKey.toString());
+  jest.setTimeout(60000);
```

Comment:
please set timeout for particular test

Reply:
For particular call?

Reply:
https://jestjs.io/docs/api#testname-fn-timeout